insights/core/__init__.py

```diff
@@ -1031,6 +1031,15 @@ class FileListing(Parser):
       directory, in the order found in the listing
     * total blocks allocated to all the entities in this directory
 
+    Note: For listings that only contain one directory, ``ls`` does not
```

Comment:
Would you mind change this `Note` format to `.. note::` (See http://www.sphinx-doc.org/en/stable/markup/para.html), sorry for the late comment.

Reply:
Ah, of course.  Hold on, I'll do that in both requests.